src/Navbar.vue

```diff
@@ -98,6 +98,13 @@ export default {
 </script>
 
 <style scoped>
+
+nav {
+  position: fixed;
+  width: 100%;
+  z-index: 1000;
+}
```

Comment:
Does that mean that now all navbar are fixed? Seems like it should be a choice that the author should be able to make, rather than imposing a fixed navbar for everyone.

Reply:
Yes, it would be good to give the choice. I don't mind the default being 'fixed' though.

Reply:
Then I would consider moving this function entirely to the MarkBind repo: probably using a jQuery.css() function to overwrite the style on `#navbar`.